battery.py

```diff
@@ -0,0 +1,67 @@
+import os
+import power
+
+battery_color_good = [  0,230,0]
+battery_color_ok   = [255,215,0]
+battery_color_bad  = [255,  0,0]
+
+BATTERY_FLASH = [None,None,None,[3],[2,3],[1,2],[2,3],[1,2],[1],None,None,None,None]
+
+def get_bat_color():
+    """
+    Function determines the color of the battery indicator. Colors can be set in config.
+    Voltage threshold's are currently estimates as voltage isn't that great of an indicator for
+    battery charge.
+    :return: false if old firmware, RGB color array otherwise
+    """
+    try:
+        v = os.read_battery()
+        if v > 3.8:
+            return battery_color_good
+        if v > 3.6:
+            return battery_color_ok
+        return battery_color_bad
+    except AttributeError:
+        return False
```

Comment:
`None` might be a better result for missing color array.

Reply:
full ack